cmd/deploy.go

```diff
@@ -365,7 +365,7 @@ func autofillApimodel(dc *deployCmd) error {
 					},
 				}
 			}
-			applicationID, servicePrincipalObjectID, secret, err := dc.client.CreateApp(ctx, appName, appURL, replyURLs, requiredResourceAccess)
+			_, applicationID, servicePrincipalObjectID, secret, err := dc.client.CreateApp(ctx, appName, appURL, replyURLs, requiredResourceAccess)
```

Comment:
@shrutir25 I think it would be good to avoid underscore placeholders since it goes against the tenets of idiomatic Go. We could change the return type of CreateApp to autorest.Response instead.

I see two benefits

1. We address the concern above.

2. It also becomes consistent with DeleteApplication.